### ISSUE TITLE
This bug occur.

### ISSUE THREAD

**Issue by chinakook** [mjq11302010044/RRPN_pytorch#2]
https://github.com/mjq11302010044/RRPN_pytorch/blob/35fc6850963a4d0ba388f25528e1d8fc2e0773c1/maskrcnn_benchmark/structures/rboxlist_ops.py#L131

**Comment by mjq11302010044** [comment]
@chinakook  If you pass a r-box in normal size ( > 16*16), it may not produce a bug

**Comment by chinakook** [comment]
My origin r-boxes are all greater than 16*16, can it be caused by your random transforms?

**Comment by mjq11302010044** [comment]
@chinakook nope, that may not happen if you have a right setting. Would you reproduce and provide your bug description?

**Comment by chinakook** [comment]
I have nearly 1200 samples, but I got this error on 15000 iters around.

**Comment by mjq11302010044** [comment]
@chinakook , Welp, I mean when your bug occurs, what's your r-box x, y, w, h and θ?

**Comment by mjq11302010044** [comment]
@chinakook  And is there any warning or error of your program?

**Comment by chinakook** [comment]
My dataset is correctly trained on RRPN(tensorflow) and R2CNN(tensorflow) etc.
No error, loss is nan after the bug occured.

**Comment by chinakook** [comment]
When I comma out ```T.RandomRotation```, this bug does not occur until now.

**Comment by mjq11302010044** [comment]
@chinakook Not really, the overlaps >1 case comes from the extremely small area of an rbox(i.e. < 1e-10).  Your Nan loss may still come from your extremely small rbox.

**Comment by chinakook** [comment]
I think maybe I have a small rbox, but ```T.RandomRotation``` make it smaller(<16*16) even.

**Comment by jxingm** [comment]
Hi, I got this error,too. Have you figured it out? Does small box cause this error?

**Comment by xuzhawei** [comment]
So the framework can not detect text that size is less than 16*16, right?

**Comment by oceanleftsea** [comment]
I start training on ICDAR15 dataset,and I also encounter this error.
